corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js

```diff
@@ -677,6 +678,28 @@ cloudCare.AppView = Backbone.View.extend({
         data.onload = function (adapter, resp) {
             cloudCare.dispatch.trigger("form:ready", form, caseModel);
         };
+        data.answerCallback = function(sessionId) {
+
+            var rawBaseUrl = self.options.urlRoot;
```

Comment:
:+1: to this though if that looks insanely hard I wouldn't necessarily block on it. (though would support @benrudolph blocking on it if he wanted)